hieradata/class/backend.yaml

```diff
@@ -51,7 +51,6 @@ govuk::node::s_base::apps:
   - canary_backend
   - collections_publisher
   - contacts
-  - content_register
```

Comment:
I don't think you want to do this just yet - it needs to stay in here for the `ensure => absent` to take effect.